gulpfile.js

```diff
@@ -2,7 +2,7 @@ const gulp = require('gulp');
 const del = require('del');
 const log = require('fancy-log');
 const gulpFilter = require('gulp-filter');
-const uglify = require('gulp-uglify');
+const uglify = require('gulp-uglify-es').default;
```

Comment:
This is unnecessary because uglify is only used for third-party libraries, which are already compiled to ES5.